src/schema/v2/me/update_collector_profile.ts

```diff
@@ -37,7 +40,13 @@ export default mutationWithClientMutationId<any, any, ResolverContext>({
       type: GraphQLString,
     },
   },
-  outputFields: CollectorProfileFields,
+  outputFields: {
+    ...CollectorProfileFields,
+    collectorProfile: {
+      type: CollectorProfileType,
+      resolve: (collectorProfile) => collectorProfile,
+    },
```

Comment:
**Questions:**
- What is the reason behind not using the existing `CollectorProfileType` for the `outputFields`?
- I also noticed that this isn't using the correct `typeOrError` along with the success/error type (e.g., [PartnerOfferType](https://github.com/artsy/metaphysics/blob/afe483696f9be14dba60e4e0ef8980ca78f91e79/src/schema/v2/createPartnerOfferMutation.ts#L70)). Implementing this in the PR shouldn't be a heavy lift, and it would help maintain consistency across our mutations.

It doesn’t feel right to me to use `...collectorProfileFields` and name the `collectorProfile` as `node` here. Can you give more context?

Reply:
> What is the reason behind not using the existing CollectorProfileType for the outputFields?

I think it's going to break Force since it uses it, but I see only a [clientMutationId](https://github.com/artsy/force/blob/main/src/Components/Inquiry/Hooks/useUpdateCollectorProfile.ts#L32) being used, so I think it's a no harm issue

>I also noticed that this isn't using the correct typeOrError along with the success/error type (e.g., [PartnerOfferType](https://github.com/artsy/metaphysics/blob/afe483696f9be14dba60e4e0ef8980ca78f91e79/src/schema/v2/createPartnerOfferMutation.ts#L70)). Implementing this in the PR shouldn't be a heavy lift, and it would help maintain consistency across our mutations

Agree, it seemed too much since I'm improving something unrelated to the work we are doing in Eigen, but once I introduce it there it's going to be stitched to an app version, making improvements later hard to be done. So, on it 🫡

Reply:
done in https://github.com/artsy/metaphysics/pull/5917/commits/455f322ce6817945b581404ca99613f7e465fa05

Reply:
:chef-kiss:

Reply:
Great to see removing the amount of duplicated lines in the schema!

<img width="354" alt="image" src="https://github.com/user-attachments/assets/64a3f4cc-1453-45e8-86e0-558c44ce3c3c">